mesh/meshdb.go

```diff
@@ -318,6 +332,36 @@ func (m *MeshDB) getLayerMutex(index types.LayerID) *layerMutex {
 	return ll
 }
 
+func getRewardKey(l types.LayerID, account types.Address) []byte {
+	str := "reward_" + account.String() + "_" + strconv.FormatUint(l.Uint64(), 10)
+	return []byte(str)
+}
+
+func getRewardSearchKey(account types.Address) []byte {
+	str := "reward_" + account.String()
+	return []byte(str)
+}
+
+func getTransactionOriginKey(l types.LayerID, t *types.Transaction) []byte {
+	str := "a_orig_" + t.Origin().String() + "_" + strconv.FormatUint(l.Uint64(), 10) + "_" + t.Id().String()
+	return []byte(str)
+}
+
+func getTransactionDestKey(l types.LayerID, t *types.Transaction) []byte {
+	str := "a_dest_" + t.InnerTransaction.Recipient.String() + "_" + strconv.FormatUint(l.Uint64(), 10) + "_" + t.Id().String()
+	return []byte(str)
+}
+
+func getTransactionOriginKeyPrefix(l types.LayerID, account types.Address) []byte {
+	str := "a_orig_" + account.String() + "_" + strconv.FormatUint(l.Uint64(), 10)
+	return []byte(str)
+}
+
+func getTransactionDestKeyPrefix(l types.LayerID, account types.Address) []byte {
+	str := "a_dest_" + account.String() + "_" + strconv.FormatUint(l.Uint64(), 10)
+	return []byte(str)
+}
+
```

Comment:
It's a matter of style, but I think that `fmt.Sprintf()` would make this much more readable.

Reply:
I agree that in some cases it is more readable but this time it will result in "a_dest_%s_%s", which is not that readable as well